jsmvc/math_components/latex_editor/latex_editor.js

```diff
@@ -113,7 +113,7 @@ define([
                     this.mathFieldAPI = MathQuill.MathField(elm[0], {
                         // todo: Options duplicated in widgets/math_input.js
                         autoCommands: 'pi and or lim',
-                        autoCommandsMapping: {'and': 'andword', 'or': 'orword'},
+                        hoMapping: {'and': 'andword', 'or': 'orword'},
```

Comment:
**question** What does this name mean? It looks like a typo. Is it a necessary change.